docs/src/main/sphinx/sql/merge.md

```diff
@@ -31,28 +31,32 @@ WHEN NOT MATCHED [ AND condition ]
 Conditionally update and/or delete rows of a table and/or insert new
 rows into a table.
 
-`MERGE` supports an arbitrary number of `WHEN` clauses with different
-`MATCHED` conditions, executing the `DELETE`, `UPDATE` or `INSERT`
-operation in the first `WHEN` clause selected by the `MATCHED`
-state and the match condition.
-
-For each source row, the `WHEN` clauses are processed in order.  Only
-the first first matching `WHEN` clause is executed and subsequent clauses
-are ignored.  A `MERGE_TARGET_ROW_MULTIPLE_MATCHES` exception is
+Data changes are performed in the `target_table` with data from the
```

Comment:
I wonder if this would this be better?

`MERGE` changes data in the `target_table` based on the contents of the `source_table`.

Reply:
Agreed. Done.